src/containers/FormikForm/components/StatusSelect.tsx

```diff
@@ -5,71 +5,74 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-import { useCallback, useEffect, useMemo, useState } from "react";
+import { useEffect, useMemo } from "react";
 import { useTranslation } from "react-i18next";
-import { Select, SingleValue, Option } from "@ndla/select";
+import { createListCollection } from "@ark-ui/react";
+import { SelectContent, SelectLabel, SelectPositioner, SelectRoot, SelectValueText } from "@ndla/primitives";
 import { IStatus as DraftStatus } from "@ndla/types-backend/draft-api";
+import { GenericSelectItem, GenericSelectTrigger } from "../../../components/abstractions/Select";
 import { PUBLISHED } from "../../../constants";
 import { ConceptStatusStateMachineType, DraftStatusStateMachineType } from "../../../interfaces";
 
 interface Props {
-  status: SingleValue;
-  setStatus: (s: SingleValue) => void;
-  onSave: (s: SingleValue) => void;
+  status: string | undefined;
+  setStatus: (s: string | undefined) => void;
+  onSave: (s: string | undefined) => void;
```

Comment:
Kan denne være undefined egentlig? 🤔

Reply:
`status` kan være undefined. `setStatus` vil aldri kalles med undefined, men la det inn for å holde ting litt likt. `onSave` kan i teorien kalles med undefined, dersom man plutselig bestemmer seg for at status skal kunne cleares